autogen.pl

```diff
@@ -979,6 +979,26 @@ sub patch_autotools_output {
 
         # We have to change the search pattern and substitution on each
         # iteration to take into account the tag changing
+        my $search_string = '# Lahey Fortran 8.1.\n.*\n\s+' .
```

Comment:
i checked and each ```$search_string``` matches a different expression.
fwiw, the new ```$search_string``` is used to insert ```flang``` related stuff, whereas the previous one is used to enhance Lahey Fortran support.~